src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -92,6 +94,14 @@ public Response getZipDownloadLimit() {
         return ok(zipDownloadLimit);
     }
 
+    @GET
+    @Path("exportFormats")
+    public Response getExportFormats() {
+        JsonArrayBuilder responseModel = Json.createArrayBuilder();
```

Comment:
Did you consider returning an object instead? Just this week I pushed a commit to return MPCONFIG options via API and I played with returning an array but ended up liking object better. Here's the code: https://github.com/IQSS/dataverse/pull/10694/commits/771d85a79a8c102d6e6b4e324c95d35d833ef40a

I'm suggesting something like this:

```
{
  "status": "OK",
  "data": {
    "OAI_ORE": {
      "displayName": "OAI_ORE"
    },
    "oai_dc": {
      "displayName": "Dublin Core"
    }
  }
}
```

Alternatively, perhaps you could add a second endpoint for getting info about a single export format (e.g. /api/info/exportFormats/oai_dc). That way you can easily get info on that format instead of having to search through an array to find the element that matches `oai_dc`.

Just some thoughts! 😄 Maybe the array is actually better.

Reply:
Yeah an object is probably better, especially with the increased amount of properties, so I switched this in [cc99729](https://github.com/IQSS/dataverse/pull/10739/commits/cc99729e6480193468e7308fe52a478ec5edb8ec)